shell/src/main/java/tachyon/shell/TfsShell.java

```diff
@@ -46,6 +47,12 @@
 @NotThreadSafe
 public class TfsShell implements Closeable {
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
+  private static final HashMap<String, String[]> CMD_ALIAS = new HashMap<String, String[]>() { {
+      put("chgrpr", new String[] {"chgrp", "-R"});
```

Comment:
now that I read the associated JIRA, I understand better ... I would not worry about backwards compatibility for these since they were introduce after last release ... but I guess we need this logic anyhow for lsr and rmr, which have been around for a while ...

I would print a WARNING for the chgrpr, chownr, chmodr commands that they are using a deprecated API (same for lsr and rmr, which should be addressed before the release) so that we can discontinue support for the *r operations in the next or the next next version